Dockerfile

```diff
@@ -0,0 +1,52 @@
+FROM python:3.9-slim-buster as builder
+
+WORKDIR /usr/src/app
+
+# set environment variables
+ENV PYTHONDONTWRITEBYTECODE 1
+ENV PYTHONUNBUFFERED 1
+
+# install system dependencies
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends gcc
+
+# lint
+RUN python -m pip install --upgrade pip
+COPY . /usr/src/app/
+RUN python -m pip install black==23.7.0
```

Comment:
You don't need to run lint in docker. It's just a waste of time. You can run it in the pipeline if you want to keep the code clean.